<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>principal_ihm</class>
 <widget class="QMainWindow" name="principal_ihm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ecosysteme</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="conteneur" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bouton_gen">
       <property name="text">
        <string>Générer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_pas">
       <property name="text">
        <string>Un pas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_sim">
       <property name="text">
        <string>Simuler</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_qui">
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bouton_qui</sender>
   <signal>clicked()</signal>
   <receiver>principal_ihm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuitter</sender>
   <signal>activated()</signal>
   <receiver>principal_ihm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
